### ISSUE TITLE
Is this app missing enable/disable custome DNS for custome network (Wifi, 4G, ...)?

### ISSUE THREAD

**Issue by Fyne5** [celzero/rethink-app#1100]
Dear authors team,

I can not found this feature in your app. So, does my device is issue and no support this feature?

Thank so much.

**Comment by ignoramous** [comment]
Yes, this feature is missing. We will see if we can implement it in one of the upcoming releases. Most of the plumbing is already there...

**Comment by ignoramous** [comment]
#1130 / #1219 / #759
